OpenSource/checklists/01-set-up.md

```diff
@@ -10,11 +10,18 @@
 - [ ] Website features a project description of less than 100 words
 - [ ] Project description is consistent across platforms
 
-## Onboarding
+## Naming and Presentation
```

Comment:
This section is identical as the one below ion line 21 (28), remove this or the other one.